```
In file included from /home/runner/_work/kokkos/kokkos/kokkos/containers/src/impl/Kokkos_UnorderedMap_impl.cpp:21:
/home/runner/_work/kokkos/kokkos/kokkos/containers/src/Kokkos_UnorderedMap.hpp:85:9: error: unused type alias 'alloc_prop_t' [-Werror,-Wunused-local-typedef]
  using alloc_prop_t = Impl::remove_cvref_t<decltype(alloc_prop)>;
        ^
1 error generated when compiling for gfx90a.
```

--- a/containers/src/Kokkos_UnorderedMap.hpp
+++ b/containers/src/Kokkos_UnorderedMap.hpp
@@ -82,8 +82,6 @@ auto allocate_without_initializing_if_possible(
 template <typename ViewType, typename... P, typename... Args>
 auto allocate_with_sequential_host_init_if_possible(
     const Impl::ViewCtorProp<P...> &alloc_prop, Args &&...args) {
-  using alloc_prop_t = Impl::remove_cvref_t<decltype(alloc_prop)>;
-
   // if incompatible we don't add the property
   if constexpr (!SpaceAccessibility<
                     typename ViewType::execution_space::memory_space,

